superset-frontend/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -216,7 +216,13 @@ class FilterBox extends React.PureComponent {
       const color = 'lightgrey';
       const backgroundImage = `linear-gradient(to right, ${color}, ${color} ${perc}%, rgba(0,0,0,0) ${perc}%`;
       const style = { backgroundImage };
-      return { value: opt.id, label: opt.id, style };
+      let label = opt.id;
+      if (label === true) {
+        label = t('True');
+      } else if (label === false) {
+        label = t('False');
```

Comment:
I'm actually not sure we want translations here (I know, goes against my usual comments!).

The reason being, we want to display the data as it exists in the table, and I don't know of any SQL dialects that translate the keywords for boolean values. I think `"TRUE"` and `"FALSE"` are probably best here, although would also accept `"true"` and `"false"`

Reply:
I thought it made sense to translate as the FilterBox is a user-facing interface, but I agree maybe more data-savvy users would want them to display as what the database displays.

I'm going to extra them as constants so that customized Superset could easily override.